util/fhir.ts

```diff
@@ -37,6 +39,14 @@ export function createPatientResourceString(birthDate: string): string {
   return JSON.stringify(pt, null, 2);
 }
 
+export function getFhirResourceString(resource: fhir4.Resource) {
```

Comment:
The name of this function seems to imply it returns the string for the entire resource. This kind of returns something like a summary or title for the resource. Also since this is a reusable function, add JSDoc comments to explain it.

Reply:
Does `getFhirResourceSummary` sound better?

Reply:
I agree with Hoss's comment. We should rename this function to something that indicates that this returns a title/summary/code and display string.